server/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const db = require('./db');
+const app = express();
+const port = 3000;
+
+
+app.get('/items/:itemId/reviews', (req, res) => {
+  db.getAllReviewsForItem(1, (err, data) => {
+    if (err) {
+      console.log('error getting item reviews: ', err);
```

Comment:
you should send a failure response here, such as `res.status(404).end()`, prior to throwing the error, or the GET origin will continue to wait.

You can test this by sending a GET request for a product that doesn't exist. Postman is great for this.

Reply:
thanks definitely forgot to do that